contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -378,32 +379,34 @@ func ChangePodStatus(newStatus, currentStatus string) {
 [lowerCamelCase]: https://en.wiktionary.org/wiki/lowerCamelCase
 [alphanumeric]: https://en.wikipedia.org/wiki/Alphanumeric
 
-### Use `klog.KObj` and `klog.KRef` for Kubernetes objects
+## Ensure value is properly passed
 
-As part of structured logging migration we want to ensure that kubernetes objects references are consistent within the
-codebase. Two new utility functions were introduced to klog `klog.KObj` and `klog.KRef`. Any reference
-(name, uid, namespace) to Kubernetes Object (Pod, Node, Deployment, CRD) should be rewritten to utilize those functions.
-In situations when object `UID` is would be beneficial for log, it should be added as separate field with `UID` suffix.
+When passing a value for a key-value pair, please use following rules:
+* Prefer using Kubernetes objects (for example `*v1.Pod`) and log them using `klog.KObj`
+  * When the original object is not available, use `klog.KRef` instead
+* Pass structured values directly (avoid calling `.String()` on them first)
```

Comment:
* When the goal is to log a `[]byte` array as string, explicitly convert with `string(<byte arrary>)`.

Reply:
done